_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,123 @@
+working with two incredible editors that have been crucial in supporting peer 
+review: David Nicholson and Ivan Ogasawara. In the upcoming months we will 
+recruit more editors to support our open source software review process. 
+Finally, I am also continuing to coordinate with and learn from the 
+well-established and successful rOpenSci. rOpenSci has set the precedent for 
+this type of organization in this space and I am grateful for their ongoing 
+support. 
+
+## pyOpenSci is not just peer review of open source software 
+Peer review is critical to the pyOpenSci mission but have bigger and broader 
+aspirations too. There are several organization 
+goals supported by the Sloan Foundation grant that we will be working on over 
+the next few years:
+* Create and grow a diverse and supportive community around a dynamic and 
+open peer review process and Python open science skills. Diversity is important and we plan to bake diversity focused programs into all of pyOpenSci activities from the beginning.
+* Streamline and improve the quality of smaller (think api wrappers, spatial data tools and domain specific tools rather than Numpy and Pandas) Python Scientific Open Source software tools that support scientific workflows.   
+  * We do this through peer review which assess not only code but and usability, documentation and ensures alignment with community accepted standards for packaging.
+  * Our peer review processing considers usability of the tools which includes documentation and “vignettes” that showcase tool functionality and provide easy quickstarts for users. 
+  * Once tools are accepted into our ecosystem, we will track tool maintenance and find new maintainers, or support sunsetting as it makes sense to ensure users aren't left hanging.
+  * Through peer review we will also address redundancy in tool functionality across the scientific python ecosystem.
+* Create a model to support peer review within domain specific subcommunities. We will begin our work here with the Pangeo community. Here, we will support a peer review process with community-specific standards applied on top of ours to ensure tool consistency.   
+* Provide training and mentorship in skills needed to contribute to open source. These skills are not just for open source. They are also fundamental to good open science practices and will support career trajectories in open science too. 
+* Grow support for maintainers. A critical component of this effort will be a communication campaign around the importance of open source to science. 
+
+### We will also continue to curate our partnership with the Journal of Open Source Software (JOSS). 
+Package maintainers can still submit a package to us if they wish to publish with 
+JOSS. Our partnership supports a "fast track" review in JOSS if its functionality 
+fits their scope and requirements. Once a package is accepted by pyOpenSci , if 
```

Comment:
remove extra space before comma:
"accepted by pyOpenSci, if "

Reply:
i rewrote that little bit it was awkward!!